_posts/2015-11-06-try-new-roi-features-on-the-demo-app.markdown

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title:  "Try new ROI features on the demo app"
+date:   2015-11-06
+---
+
+As you may have heard, we are currently adding ROI support to OMERO.figure, allowing
+you to draw ROIs on image panels and copy & paste these between images.
+This post describes the new features, invites you to try them out on the demo app
+and provide feedback ahead of their inclusion in the next release. 
+
+<h2>Adding ROIs to image panels</h2>
+
+<img src="https://cloud.githubusercontent.com/assets/900055/9478881/5894a230-4b72-11e5-91c7-6ac2237a0a67.png" 
+  style="width:600px"/>
+
+To add ROIs to panels in the figure, select a single panel, click on the "Labels" tab
+and then click on the "Draw" button under the ROIs header.
+This will launch the ROI viewer, where you can draw, edit and delete a number of ROI
+types; Currently we support Rectangles, Lines, Arrows and Ellipses. 
+
+Drawing and editing of ROIs should "just work" as you expect it to: Select the type
+of ROI in the toolbar, as well as the line width and colour. Then draw the ROI on the image.
+You can also use the "select" arrow to select and edit existing ROIs.
+
+You can also copy, paste and delete ROIs using the "Edit" menu options or keyboard shortcuts.
```

Comment:
Second sentence in a row with 'also' in it. Maybe change to "You can also use the Select arrow to select and edit existing ROIs or copy, paste and delete ROIs using the Edit menu options ot keyboard shortcuts."